File: 1768. Merge Strings Alternately.java
class Solution {
    public String mergeAlternately(String word1, String word2) {
        StringBuilder result = new StringBuilder();
        int len = Math.min(word1.length(), word2.length());
        for(int i = 0; i < len; i++) {
            result.append(word1.charAt(i));
            result.append(word2.charAt(i));
        }
        result.append(word1.substring(len)).append(word2.substring(len));
        return result.toString();
    }
}

File: 1768. Merge Strings Alternately.js
function mergeAlternately(word1, word2) {
    let result = '';
    let len = Math.min(word1.length, word2.length);
    for(let i = 0; i < len; i++) {
        result += word1[i] + word2[i];
    }
    result += word1.slice(len) + word2.slice(len);
    return result;
}
